sfs/array.py

```diff
@@ -484,6 +560,15 @@ def weights_midpoint(positions, closed):
     (N,) numpy.ndarray
         Weights of secondary sources.
 
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        x0, n0, a0 = sfs.array.circular(2**8, 1)
+        a = sfs.array.weights_midpoint(x0, closed='True')
```

Comment:
`'True'` happens to be `True`, but you should still use `closed=True`.

Reply:
old habits, changed to True